platform/viewer/src/connectedComponents/ToolContextMenu.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { commandsManager } from './../App.js';
 
-import './ToolContextMenu.css';
+import { ContextMenu } from '@ohif/ui';
```

Comment:
Do we want `ToolContextMenu` to appear on right click of annotations if the `MeasurementExtension` isn't registered? If yes, then we can keep it here. If not, then we should shift this component to the measurement's extension and pass in `CommandsManager` from a `getModuleFn`

Reply:
I think this task: https://github.com/OHIF/Viewers/issues/1263 can handle this right?